Sources/TuistCore/ContentHashing/HashingFilesFilter.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+import TSCBasic
+
+internal class HashingFilesFilter {
```

Comment:
nit: I'd prefer avoiding declaring something `internal` since it's implicit, for example line 14 is `internal` as well

Reply:
That's a broader discussion. While some people prefer using the modifier only when needed, Apple recommends in the [stdlib to always be explicit when using internal](https://github.com/apple/swift/blob/a73a8087968f9111149073107c5242d83635107a/docs/AccessControlInStdlib.rst#internal).

I can do whatever is preferred here in the repo.

Reply:
it's really minor, I'm fine with both approaches, but it would be better to be consistent at least at a class level?

Reply:
Fine by me. Will remove them.